mods/ts/rules/aircraft.yaml

```diff
@@ -442,6 +446,7 @@ APACHE:
 		Speed: 130
 		CanSlide: false
 		TakeOffOnResupply: true
+		IdleBehavior: Land
```

Comment:
Leftover from a test commit?

Reply:
yes

Reply:
we can just ignore the test commit when merge

Reply:
You can remove it now.

Reply:
Removed and rebased to a newer head